include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -693,6 +693,60 @@ def TTIR_UnsqueezeOp : TTIR_DPSOp<"unsqueeze"> {
     let hasVerifier = 1;
 }
 
+def TTIR_ClampOp : TTIR_DPSOp<"clamp"> {
+    let summary = "Clamp op.";
+    let description = [{
+      Clamp tensor values to a specified range.
+
+      Example:
+        min: 2.000000+00
+        input: [[0, 1, 2, 3, 4, 5, 6, 7]]
+        max: 5.000000+00
```

Comment:
Hm, interesting. I had the same problem with `power` op https://github.com/tenstorrent/tt-mlir/issues/1094.

@mrakitaTT since `stablehlo.clamp` defines args like this

```
  let arguments = (ins
    HLO_Tensor:$min, /*clamp_i1*/
    HLO_Tensor:$operand, /*clamp_c3, clamp_i2*/
    HLO_Tensor:$max /*clamp_i3*/
  );
  let results = (outs HLO_Tensor:$result);
```

should `TTIR` have 1:1 matching arg interface or is 

```
    let arguments = (ins AnyRankedTensor:$input,
                         AnyRankedTensor:$output,
                         F32Attr:$min,
                         F32Attr:$max,
                         TT_OperandConstraintArrayAttr:$operand_constraints);
```

ok, considering what Asif said above?